test/report/test-report-exclude-network.js

```diff
@@ -38,4 +38,22 @@ describe('report exclude network option', () => {
     const report = process.report.getReport();
     assert.strictEqual(report.header.networkInterfaces, undefined);
   });
+
+  it('should not do DNS queries in libuv if exclude network', async () => {
+    await fetch('http://127.0.0.1');
+
+    process.report.excludeNetwork = false;
+    let report = process.report.getReport();
+    let tcp = report.libuv.find((uv) => uv.type === 'tcp');
+    assert.notEqual(tcp, null);
+    assert.strictEqual(tcp.localEndpoint.host, 'localhost');
+    assert.strictEqual(tcp.remoteEndpoint.host, 'localhost');
+
+    process.report.excludeNetwork = true;
+    report = process.report.getReport();
+    tcp = report.libuv.find((uv) => uv.type === 'tcp');
+    assert.notEqual(tcp, null);
+    assert.strictEqual(tcp.localEndpoint.host, '127.0.0.1');
+    assert.strictEqual(tcp.remoteEndpoint.host, '127.0.0.1');
+  });
```

Comment:
Does this have to go in `test/internet`?

Reply:
Good question, it's still a test for getReport and the request is self contained to localhost, so does it make sense in internet?

Reply:
I don't know where it belongs, we'll see if someone else mentions it.

Reply:
I don't think this feature access the "internet". It just accesses local network information.